decrypt/decrypt.go

```diff
@@ -90,16 +94,35 @@ func Decrypt(args []string) error {
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
 			}
+			decryptedCount++
 		case helpers.FileExists(file.Unencrypted):
 			fmt.Fprintf(os.Stderr, "Warning: file %s is already decrypted, skipping\n", file.Unencrypted)
 		default:
 			fmt.Printf("Decrypting file %v/%v: %s\n", i+1, numFiles, file.Encrypted)
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
```

Comment:
```suggestion

				continue
```

Reply:
Why would we count a failed decryption? I think a user should know in the summary that some of the attempts failed. Otherwise we can also use `numFiles` and ditch decryptedCount

Reply:
The count will never happen on an error since the `continue` will force it to restart on the next file in the list.

Reply:
sorry, morning brain, the continue part didn't register to me.